vm/devices/get/guest_emulation_transport/src/process_loop.rs

```diff
@@ -1853,7 +1853,10 @@ async fn request_igvm_attest(
     };
 
     let response_length = response.length as usize;
-    if response_length > ALLOCATED_SHARED_MEMORY_SIZE {
+    if response_length == get_protocol::HOST_VMWP_ATTESTATION_GENERIC_ERROR_CODE {
+        tracing::error!("Agent returned an error");
```

Comment:
Should this message be updated as well like you did for `IgvmAgentGenericError`?

Reply:
This message does not actually show up as part of the span.  maybe there is bug in here, theoretically it should show up as part of the span.  Removing it.